### ISSUE TITLE
Nodes are broken in 2.0.X

### ISSUE THREAD

**Issue by JeromeCHA** [SceneView/sceneview-android#379]
Hi, as I mentioned in the title, after I updated to 2.0.X, it looks like SphereNode is broken.
(confirmed it also happens in `2.0.2`)

Here is the sample code.

```kotlin
        binding.sceneView.apply {
            SphereNode(
                engine = engine,
                radius = 1f
            ).also {
                addChildNode(it)
            }
        }
```

Produces the following result :

| 1.2.6 | 2.0.X |
|:-:|:-:|
| ![Screenshot_20231225_154649](https://github.com/SceneView/sceneview-android/assets/11457628/4731e886-5c36-4dd4-8b4f-c9a9fb3163df) | ![Screenshot_20231225_154724](https://github.com/SceneView/sceneview-android/assets/11457628/80b0830f-1f6f-4d5b-8a7c-72ca0b5df19f) |

Am I doing something wrong or did something change between the two versions for Nodes? 🤔

**Comment by JeromeCHA** [comment]
Same with PlaneNode

| 1.2.6 | 2.0.X |
|:-:|:-:|
| ![Screenshot_20231227_134320](https://github.com/SceneView/sceneview-android/assets/11457628/17593b85-8ae7-4f87-a1f7-57016579cc1e) | ![Screenshot_20231227_134109](https://github.com/SceneView/sceneview-android/assets/11457628/0ba8b8d4-b320-40e3-bf36-7a0b035b63fd) |

**Comment by Mehequanna** [comment]
I am also seeing this broken behavior.

**Comment by nallaperumal** [comment]
Thanks for the reporting as I am also facing the same issue. Please also help me if we can create a textnode to place text content and any sample code for the same.

**Comment by ryust** [comment]
Was this "fixed" in 2.0.3 with #392? I am still experiencing similar shearing of models in the latest release.
